python/setup.py

```diff
@@ -208,8 +208,12 @@ def _run_cmake(self):
         def move_lib(lib_name):
             lib_filename = (shared_library_prefix + lib_name +
                             shared_library_suffix)
-            shutil.move(pjoin(self.build_type, lib_filename),
-                        pjoin(build_lib, 'pyarrow', lib_filename))
+            # Also copy libraries with ABI/SO version suffix
+            libs = glob.glob(pjoin(self.build_type, lib_filename) + '*')
+            for lib in libs:
+                filename = os.path.basename(lib)
+                shutil.move(pjoin(self.build_type, filename),
+                            pjoin(build_lib, 'pyarrow', filename))
```

Comment:
Some of these are symlinks, though, right? is there a way to only move the non-symlinked libs?